scripts/keras_benchmarks/data_generator.py

```diff
@@ -0,0 +1,35 @@
+""" Generates input and label data for training models. """
+import numpy as np
+
+
+def generate_img_input_data(input_shape, num_classes):
+  """Generates training data and target labels.
+
+  # Arguments
+    input_shape: input shape in the following format
+                      (num_samples,channels, x, y)
+    num_classes: number of classes that we want to classify the input
+
+  # Returns
+    numpy arrays: 'x_train, y_train'
+  """
+  x_train = np.random.randint(0, 255, input_shape)
+  y_train = np.random.randint(0, num_classes, (input_shape[0],))
+
+  return x_train, y_train
+
+
+def generate_text_input_data(input_shape):
+  """Generates training data and target labels.
+
+  # Arguments
+    input_shape: input shape in the following format(num_samples, x, y)
+
+  # Returns
+    numpy arrays: 'x_train, y_train' where the value of the arrays are booleans.
+  """
+  x_train = np.random.choice([True,False], input_shape)
```

Comment:
I don't fully understand how this is analogous to generating text data?

I think more appropriate would be `np.random.binomial(2, p, input_shape)` where `p` is chosen to match the ratio of words from the vocabulary found in the average sample (e.g. p=0.05 if each sample contains on average 5% of all words available). But then it would be good to mention that this is meant to model text where each sample is encoded as a single vector with binary encoding of word presence

Reply:
Should actually be `np.random.binomial(1, p, input_shape)`, sorry.